services/frontend/src/utils/models/Project.js

```diff
@@ -86,6 +86,17 @@ export default class Project {
             return project;
         });
     }
+
+    static getContributors(profilesById = {}, projectProfilesForProject = {}) {
+        return projectProfilesForProject.reduce((acc, projectProfile) => {
+            const contributorProfile = {...projectProfile};
```

Comment:
I suggest creating a new utility model called `Contributor` so that we can have a defined interface for how exactly we're building these contributor objects. That is, it would just be a basic data model with a `profile`, `role`, and auxiliary attributes. Alternatively, it _could_ flatten the properties of the `profile` onto itself, but I leave that decision up to you.

This would help to solidify the API between what's coming from the store and what's being used in the components.